packages/common/src/pipes/date_pipe.ts

```diff
@@ -10,14 +10,51 @@ import {Inject, InjectionToken, LOCALE_ID, Optional, Pipe, PipeTransform} from '
 
 import {formatDate} from '../i18n/format_date';
 
+import {DatePipeConfig, DEFAULT_DATE_FORMAT} from './date_pipe_config';
 import {invalidPipeArgumentError} from './invalid_pipe_argument_error';
 
 /**
  * Optionally-provided default timezone to use for all instances of `DatePipe` (such as `'+0430'`).
  * If the value isn't provided, the `DatePipe` will use the end-user's local system timezone.
+ *
+ * @deprecated use DATE_PIPE_DEFAULT_OPTIONS token to configure DatePipe
  */
 export const DATE_PIPE_DEFAULT_TIMEZONE = new InjectionToken<string>('DATE_PIPE_DEFAULT_TIMEZONE');
 
+/**
+ * DI token that allows to provide default configuration for the `DatePipe` instances in an
+ * application. The value is an object which can include the following fields:
+* - `dateFormat`: configures the default date format. If the not provided, the `DatePipe`
+ * will use the 'mediumDate' as a value.
+ * - `timezone`: configures the default timezone. If the value isn't provided, the `DatePipe` will
```

Comment:
(for consistency with the previous sentence)
```suggestion
 * - `timezone`: configures the default timezone. If not provided, the `DatePipe` will
```